From 1258a99decf96092de8ea512150209e14152ed9c Mon Sep 17 00:00:00 2001
From: Craig Andrews <candrews@integralblue.com>
Date: Tue, 30 Jul 2019 16:46:20 -0400
Subject: [PATCH] Install AmdCompiler.h header

ROCm-OpenCL-Runtime uses AmdCompiler.h so it should be installed to the system and made available for use
---
 src/driver/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/driver/CMakeLists.txt b/src/driver/CMakeLists.txt
index 55ed13f..8f68fb7 100644
--- a/src/driver/CMakeLists.txt
+++ b/src/driver/CMakeLists.txt
@@ -91,3 +91,4 @@ target_link_libraries(opencl_driver ${llvm_libs})
 target_include_directories(opencl_driver PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 
 install(TARGETS opencl_driver DESTINATION ${CMAKE_INSTALL_LIBDIR} )
+install(FILES AmdCompiler.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
